apps/prairielearn/src/tests/instructorQuestions.test.sql

```diff
@@ -4,16 +4,6 @@ SELECT
 FROM
   questions;
 
--- BLOCK select_last_job_sequence
```

Comment:
This one is unrelated, but came up while I was double-checking that the query was no longer in use.